lib/dor/services/merge_service.rb

```diff
@@ -58,23 +58,25 @@ def copy_workspace_content
       end
     end
 
-    def decomission_secondaries
+    def decommission_secondaries
       @secondary_objs.each do |secondary_obj|
         begin
           @current_secondary = secondary_obj
-          @current_secondary.decomission @tag
+          @current_secondary.decommission @tag
           @current_secondary.save
 
           unshelve
           unpublish
           Dor::CleanupService.cleanup_by_druid @current_secondary.pid
           Dor::Config.workflow.client.archive_active_workflow 'dor', @current_secondary.pid
         rescue => e
-          @logger.error "Unable to decomission #{@current_secondary.pid} with primary object #{@primary.pid}: #{e.inspect}"
+          @logger.error "Unable to decommission #{@current_secondary.pid} with primary object #{@primary.pid}: #{e.inspect}"
           @logger.error e.backtrace.join("\n")
         end
       end
     end
+    alias_method :decomission_secondaries, :decommission_secondaries=
```

Comment:
I don't see where `decommission_secondaries=` is defined? should the `=` be dropped?

Reply:
Yep, it was the same copy-pasta as the other method.